api/src/pcapi/core/booking_providers/cds/client.py

```diff
@@ -205,14 +205,14 @@ def book_ticket(self, show_id: int, quantity: int) -> list[booking_providers_mod
             raise cds_exceptions.CineDigitalServiceAPIException(f"Unavailable pass culture tariff for show={show.id}")
 
         ticket_sale_collection = self._create_ticket_sale_dict(show, quantity, screen, show_voucher_type)
-        payment = self._create_transaction_payment(show_voucher_type)
+        payement_collection = self._create_transaction_payment(quantity, show_voucher_type)
```

Comment:
typo sur pay~~e~~ment

```suggestion
        payment_collection = self._create_transaction_payment(quantity, show_voucher_type)
```